packages/frontend/README.md

```diff
@@ -22,6 +22,8 @@ Now you can access the application on [http://localhost:3005](http://localhost:3
 
 > NOTE: When running Rafiki Admin in the development environment, it will connect to Cloud Nine Wallet Admin GraphQL - [http://localhost:3001/graphql](http://localhost:3001/graphql).
 
+To add a new typed apollo request, you will need to add an untyped request and regenerate the graphql types. This will generate new types tailored to the specific request being made. The generated type will reflect the request's query or mutation name, variables used, and requested fields.
+
```

Comment:
I added this note because it took me a long time to figure out how these types were generated. I figured the generated gql files were solely determined by the backend gql schema file.

Hopefully this note is clear - the instructions to generate types are above this, so I think it makes more sense in context.

Reply:
What is meant by "untyped request" here?

Reply:
By untyped I mean not passing in type arguments to the query call. So doing this:

```ts
const response = await apolloClient.query({
    query: gql`
      query GetThingQuery($id: String!) {
        thing(id: $id) {
          id
          createdAt
        }
      }
    `,
    variables: args
  })
```

Then run `pnpm --filter backend generate`.

Then update the request with the newly generated types:
```ts
const response = await apolloClient.query<GetThingQuery, GetThingQueryVariables>({ ...
```

Open to feedback if this can be clearer. My main point of confusion was that the types were being generated by the apollo requests themselves (as opposed to the scheme in the backend). This (new apollo request -> backend generate cmd -> use types) workflow was not immediately obvious to me which is why I added the note.

Reply:
Ah, I got it now, thank you! I think it would make sense while developing - I think the note is sufficient